### DISCUSSION TITLE
Does anyone have the Rotor inertia for the 750 kW wind turbine?

### DISCUSSION THREAD

**Discussion by eletroinf** [OpenFAST/openfast#1454] [Q&A]
I need this parameter for a simplified model.
I think it should be a good idea to provide the Rotor Inertia parameter in the general documentation. 
That's useful for reduced-order models where the drivetrain is modeled considering only the rotor inertia.

Regards,
Ricardo

**Comment by jjonkman** [comment]
Dear @eletroinf,

If you already have an ElastoDyn model of a 750 kW turbine within OpenFAST, ElastoDyn will integrate the blade mass distribution and report the total rotor inertia in the ElastoDyn summary file.

If you don't already have an OpenFAST model, you can find some properties of a generic 750 kW turbine in the old WindPACT Turbine Rotor Design Study report: https://www.nrel.gov/docs/fy06osti/32495.pdf.  I don't recall seeing the rotor inertia in this report, but other details are provided, which may enable you to estimate the rotor inertia.  (I'm not aware that OpenFAST models exist for each of the WindPACT turbines, except for the 1.5-MW WindPACT turbine that is shared through the OpenFAST r-test.)

Best regards,
